taxi/views.py

```diff
@@ -51,4 +52,44 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
-    queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+    queryset = Driver.objects.prefetch_related("cars__manufacturer")
+
+
+class CarsCreateView(generic.edit.CreateView):
+    model = Car
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:car-list")
+    template_name = "taxi/car_form.html"
+
+
+class CarsUpdateView(LoginRequiredMixin, generic.edit.UpdateView):
```

Comment:
```suggestion
class CarsUpdateView(LoginRequiredMixin, generic.UpdateView):
```

and same for cases bellow